docs/components/optimize/what-is-optimize.md

```diff
@@ -0,0 +1,26 @@
+---
+id: what-is-optimize
+title: What is Optimize?
+---
+
+Camunda is largely comprised of three key components:
+
+- Design
+- Automate
+- Improve
+
+A user can design process flows through tools like Cawemo and Modeler, and deploy applications via Modeler or an IDE into the server. A user can use Tasklist to review and complete tasks, and Operate to view and analyze process instances.
```

Comment:
I'm not sure I understand "and deploy applications via Modeler or an IDE into the server." In a production scenario you could deploy through Camunda Modeler, Console Modeler, or programmatically. Maybe replace "an IDE" with programmatically to cover APIs and clients? 

Also can you link to the other components' documentation mentioned?

Reply:
Yes! Updated in latest commit.